tests/test_base.py

```diff
@@ -40,7 +41,7 @@ def setUp(self):
         Runs the short simulation over reynolds mountain east
         """
 
-        self.test_dir = os.path.abspath('tests/RME')
+        self.test_dir = os.path.abspath('tests/Lakes')
```

Comment:
This method could be changed to `setUpClass`

Reply:
Agreed, it only needs to be ran once.